docs/man/crystal/README.md

```diff
@@ -354,11 +354,36 @@ crystal play [--port <port>] [--binding <host>] [--verbose] [file]
 ### `crystal tool`
 
 * `crystal tool context`: Show context for given location
+* [`crystal tool dependencies`](#crystal-tool-dependencies): Show tree of required source files
 * `crystal tool expand`: Show macro expansion for given location
 * [`crystal tool format`](#crystal-tool-format): Format Crystal files
 * `crystal tool hierarchy`: Show type hierarchy
 * `crystal tool implementations`: Show implementations for given call in location
 * `crystal tool types`: Show types of main variables
+* [`crystal tool unreachable`](#crystal-tool-unreachable): Show methods that are never called.
+
+### `crystal tool dependencies`
+
+Show tree of required source files.
+
+```
+crystal tool dependencies [options] [programfile]
+```
+
+Options:
+
+* `-DFLAG`, `--define=FLAG`: Define a compile-time flag. This is useful to
+  conditionally define types, methods, or commands based on flags available at
+  compile time. The default flags are from the target triple given with
+  `--target-triple` or the hosts default, if none is given.
+* `-fFORMAT`, `--format=FORMAT`: Output format `tree` (default), `flat`, `dot`, or `mermaid`.
```

Comment:
There's an inconsistency of the presentation for flags. Here you say `-fFORMAT` and `--format=FORMAT` and in the unreachable tool you place `-f text|json` and `--format text|json`. Same for include and exclude.